src/shared/components/palette/date/read-date-field.component.ts

```diff
@@ -1,8 +1,10 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
```

Comment:
_Nitpick:_
`OnInit` no longer needs to be imported as as the component doesn't implement the interface.

Reply:
Done